fcl/caf/cafmakerjob_icarus_detsim2d.fcl

```diff
@@ -0,0 +1,10 @@
+#include "cafmakerjob_icarus_systtools_and_fluxwgt.fcl"
```

Comment:
This is a BNB specific fhicl right -- for NuMI I don't think fluxwgt is necessary?

If I'm correct, maybe you should specify this name a bit more and make the equivalent one for NuMI?

Reply:
fluxwgt will just not run for NuMI, so I think it's fine. It's better than having to double the number of CAF configurations for BNB/NuMI.

Reply:
Ok good, was worried it would cause issues.